pkgs/development/guile-modules/guile-gcrypt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, guile, libgcrypt
+, autoreconfHook, pkgconfig, texinfo
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-gcrypt";
+  version = "0.2.1";
+
+  src = fetchurl {
+    url = "https://notabug.org/cwebber/${pname}/archive/v${version}.tar.gz";
+    sha256 = "1qj1yw0kman984x584jjjxnjdhm0pwgp09iyn3b5rqajx7svpqcd";
+  };
+
+  postConfigure = ''
+    sed -i '/moddir\s*=/s%=.*%=''${out}/share/guile/site%' Makefile;
+    sed -i '/godir\s*=/s%=.*%=''${out}/share/guile/ccache%' Makefile;
```

Comment:
Is it not possible to set these variables through environment variables or pass them to make somehow?

Reply:
No (or I would have done :p)

I know this is flimsy, but it's also a very short Makefile so there's little room for error here.